src/modules/completed.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable eqeqeq */
+
+export default function preValidate(valid, e, arr) {
+  const dataId = e.target.parentElement.getAttribute('data-id');
+
+  e.target.parentElement.setAttribute('data-valid', valid);
+
+  const todoItems = arr;
+  todoItems.find((elem) => {
+    let i;
+    if (elem.index == dataId) {
+      elem.completed = valid;
+      i = 1;
+    } else i = 0;
+    return i;
+  });
+
+  localStorage.setItem('todoItems', JSON.stringify(todoItems));
+}
```

Comment:
- [ ] Kindly note that after marking all activities as completed and clicking on the `clear completed` button,  a task is still left on the list yet all the tasks are expected to be removed. Kindly check out that bug.

### before clearing the tasks

![image](https://user-images.githubusercontent.com/98453979/220406558-6530fd65-54e7-4bce-813e-06f969b6ceee.png)

### after clearing the tasks

![image](https://user-images.githubusercontent.com/98453979/220406719-dbc86910-9a92-40a2-9c19-2fdde423b0a8.png)